src/nwb_conversion_tools/nwbconverter.py

```diff
@@ -53,14 +53,14 @@ def get_conversion_options_schema(cls):
         return conversion_options_schema
 
     @classmethod
-    def validate_source(cls, source_data):
+    def validate_source(cls, source_data: Dict[str, dict], verbose: bool = True):
```

Comment:
Should not we make this false by default?

Reply:
I'm keeping it True for now just for backcompatability